content/working_with/console/widgets/cloudNum.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: bt_wiki
+title: Number of clouds
+category: Widgets
+draft: false
+---
+Displays the cloud icon.
```

Comment:
Is that it? 🙂

Reply:
Yes 🙂 At this moment that's it.

There are plans to extend this widget to provide real number, but for 6.2 it'll probably be just it.